templates/openvcloud/openvcloud.py

```diff
@@ -11,22 +11,34 @@ def __init__(self, name, guid=None, data=None):
         super().__init__(name=name, guid=guid, data=data)
 
         self._validate_data()
+        self._configure()
 
+    def _validate_data(self):
```

Comment:
why not using the `validate` method ?

Reply:
that's a different one, that is actually called from the constructor, not to be mistaken with the `validate` method.